src/PlaneWaveBasis.jl

```diff
@@ -220,6 +220,12 @@ function PlaneWaveBasis(model::Model{T}, Ecut::Number, fft_size, variational,
     # by the same process
     n_kpt   = length(kcoords_global)
     n_procs = mpi_nprocs(comm_kpts)
+    # Custom reduction operators for MPI are currently not working on aarch64, so
+    # fallbacks are defined in common/mpi.jl. For them to work, there cannot be more
+    # than 1 MPI process.
+    if Base.Sys.ARCH == :aarch64 && n_procs > 1
+        error("MPI not supported on aarch64")
```

Comment:
Move that check to MPI init?

Reply:
By that I mean: define our own mpi_init

Reply:
`MPI.Init()` doesn't require knowledge of a specific communicator, but _we_ need to have the MPI communicator used by the k-points to check the number of processes it's using. I think that in practice, this is always `COMM_WORLD` (by default), but it doesn't have to be.
Michael had the same suggestion at first, but I'm not sure that it works out.

Reply:
For this workaround, we need comm world to have rank 1 so I think it's fine?